L/LibVPX/build_tarballs.jl

```diff
@@ -3,22 +3,25 @@
 using BinaryBuilder
 
 name = "LibVPX"
-version = v"1.10.0"
+version = v"1.15.2"
 
 # Collection of sources required to build LibVPX
 sources = [
-    ArchiveSource("https://github.com/webmproject/libvpx/archive/v$(version).tar.gz",
-                  "85803ccbdbdd7a3b03d930187cb055f1353596969c1f92ebec2db839fa4f834a"),
+    GitSource("https://github.com/webmproject/libvpx", "d168454ecd099805c675d4a98c66f4891373302a"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd ${WORKSPACE}/srcdir/libvpx-*/
+cd ${WORKSPACE}/srcdir/libvpx
 sed -i 's/cp -p/cp/' build/make/Makefile
 
-mkdir vpx_build && cd vpx_build
 apk add diffutils yasm
 
+if [[ "${bb_full_target}" == armv7l-* ]]; then
```

Comment:
I think this should really be fixed in BBB: https://github.com/JuliaPackaging/Yggdrasil/pull/11596#discussion_r2183912224

Reply:
Remove the work-around.